### ISSUE TITLE
Support for ListenBrainz

### ISSUE THREAD

**Issue by k8ieone** [ultrasonic/ultrasonic#277]
So Airsonic recently added support for ListenBrainz scrobbling. From what I understand, this also has to be supported by the client in order to work. Is it even viable to support this, because from what I looked at, only Airsonic supports ListenBrainz so this could break compatibility with other servers, right?

**Comment by ogarcia** [comment]
Ultrasonic only supports standar Subsonic API. If Airsonic has a non-standard feature we cannot implement for compatibility reasons.

**Comment by k8ieone** [comment]
I completely understand that. Thank you for considering my request.

**Comment by KingDuckZ** [comment]
I got here looking for info about LB support. I also use Airsonic so that would be compatible with my server, but regardless, the feature I'd like to ask is about Ultrasonic reporting to ListenBrainz, just like [Strawberry](https://github.com/strawberrymusicplayer/strawberry) does. My understanding is that has nothing to do with Airsonic, it's just about obtaining access to a LB account and reporting the metadata of the songs being played. Strawberry does it with both Airsonic and local music. Would it be possible to add this to Ultrasonic too?

**Comment by Maxmystere** [comment]
I think what you want is already in Ultrasonic under `Settings -> Scrobbler for Last.fm` 

I'm personally using [lms](https://github.com/epoupon/lms) and checking this checkbox enable ultrasonic reporting to ListenBrainz if server is well configured

If there is an increase in other than last.fm usage maybe checkbox could be renamed to a more generic name @ogarcia ?

**Comment by ogarcia** [comment]
@Maxmystere maybe the setting must have `Scrobbler` as label, because this setting only performs a call to http://www.subsonic.org/pages/api.jsp#scrobble API endpoint

**Comment by KingDuckZ** [comment]
Thanks for the info. I did test with the LB credentials set on my Airsonic server and saw that LB was receiving no updates, but I had that Last.fm option disabled - I would have not guessed it'd make a difference, I agree it should be renamed ("Scrobble to any configured service on the server" in the option description would be massively useful, imo).

It's not *that simple* tho :D Configuring LB on Airsonic directly seems to spam LB with multiple listens in a short time if I play something from the browser (not sure about the app), and that's probably a bug in Airsonic. Development on their end is extremely slow, so I just disabled it for now. That's why I'd prefer if Ultrasonic did that directly, instead of relying on Airsonic doing it correctly!
